charts/s3gw/values.yaml

```diff
@@ -16,33 +17,39 @@ secretKey: "test"
 ui:
   # 'enabled' will deploy the S3GW user interface if set to `true`.
   enabled: true
-  # 'hostname' is the hostname of the S3GW user interface accessible via HTTPS.
-  hostname: "s3gw-ui.local"
+  # 'serviceName' is the service name of the S3GW user interface.
+  serviceName: "s3gw-ui"
+  # 'publicDomain' is the public domain of the UI Service used by the Ingress.
+  publicDomain: "127.0.0.1.omg.howdoi.website"
 
   # TLS certificate and key
-  # Base64 encoded in one line. The CN must match .ui.hostname
+  # Base64 encoded in one line. The CN must match .ui.serviceName + .ui.publicDomain
```

Comment:
The given default certificates now don't match that since they are still signed to `s3gw.local` iirc. Do we want to change that?

Reply:
Yup, it makes perfectly sense.
Are we agreeing to use `127.0.0.1.omg.howdoi.website` as default?
Can we update certificates on another PR?

Reply:
default certificates updated.